crypto/cmp/cmp_genm.c

```diff
@@ -344,3 +344,67 @@ int OSSL_CMP_get1_rootCaKeyUpdate(OSSL_CMP_CTX *ctx,
     X509_free(oldWithOld_copy);
     return res;
 }
+
+int OSSL_CMP_get1_crlUpdate(OSSL_CMP_CTX *ctx, const X509_CRL *last_crl,
+                            X509_CRL **crl)
```

Comment:
Why don't you use a `const X509 *cert` parameter is in `CMPclient_crlUpdate()`?
This would make the function more flexible to use.